source/_posts/2024-09-03-aqara-joins-works-with-home-assistant.markdown

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
This blog is already released.

Reply:
yes, it is the merge from current into RC

Reply:
I think Franck has a different merge strategy for this PR where rc is merged into current so that only the new stuff appears and we can review the release notes in full instead of just the changes from after the beta cut.